chain/chain/src/store.rs

```diff
@@ -2489,31 +2496,33 @@ impl<'a> ChainStoreUpdate<'a> {
                 continue;
             }
 
-            match chunk_hashes_by_height.entry(chunk.header.inner.height_created) {
+            let height_created = match chunk {
```

Comment:
I thought about it, but I think this is the only place where `height_created` is directly accessed from the chunk, so it seemed unnecessary to add a method which is only used in one place. I'll switch it though if you think there is a chance it will be used in more places in the future.

Reply:
Okay we don't have to do it now. Can do the refactor when we need it again